server/models/admin.sql

```diff
@@ -0,0 +1,105 @@
+DELIMITER $$
+
+/*
+ zRecomputeEntityMap
+
+ Abolishes and recomputes the entity_map from the base tables in the system.  This is
+ useful in case of database corruption in which references get out of sync.
+*/
+CREATE PROCEDURE zRecomputeEntityMap()
```

Comment:
This looks like it will be very useful - what is the syntax and meaning for procedures following a 'z'?